Makefile

```diff
@@ -151,16 +151,21 @@ clean-glide-cache:
 deps: prebuild-check
 	$(GLIDE_BIN) --verbose install
 
-.PHONY: generate
-## Generate GOA sources. Only necessary after clean of if changed `design` folder.
-generate: prebuild-check $(DESIGNS) $(GOAGEN_BIN) $(GO_BINDATA_ASSETFS_BIN) $(GO_BINDATA_BIN)
+app/controllers.go: $(DESIGNS) $(GOAGEN_BIN)
```

Comment:
Thanks @aslakknutsen I understand how `make` targets works in deciding whether to execute or not, I was curious about what made us choose `app/controllers.go` instead of the other *.go files which too are subject to change( since they are auto-generated).